scripts/stage1.sh

```diff
@@ -81,7 +83,8 @@ mkdir -p "$img/bin/"
 
 for file in /bin/busybox /bin/kmod /bin/systemd-tmpfiles /bin/udevadm \
 	/lib/systemd/systemd-networkd /lib/systemd/systemd-udevd \
-	/usr/sbin/dropbear /usr/lib/dropbear/dropbearconvert; do
+	/usr/sbin/dropbear /usr/lib/dropbear/dropbearconvert /sbin/zfs \
+	/sbin/zpool /sbin/zdb; do
```

Comment:
Instead of having to use external debs for `zfs`, could we instead copy those utilities from the current file system in the `postinst` script like we do for the zfs kernel modules?

Reply:
We could, but generally speaking I prefer to get as much as possible from the debs whenever it is easy; the reason we don't do so for all the kernel modules is that those are architecture specific, and so would balloon the size of the environment significantly. Userland utilities and libraries don't have the same issue.  We also would need to copy the dependency resolution logic into the postinst script, which is not something we currently need.  Is there a compelling reason to do so?

Reply:
> We also would need to copy the dependency resolution logic into the postinst script, which is not something we currently need. Is there a compelling reason to do so?

Ah okay, that sounds like a compelling reason for keeping it here. I was just wondering if we could have simplified the logic, but have not thought about the dependencies.